services/core-api/app/api/mines/documents/models/mine_document.py

```diff
@@ -37,6 +38,12 @@ class _ModelSchema(Base._ModelSchema):
     archived_by = db.Column(db.String(60))
 
     versions = db.relationship('MineDocumentVersion', lazy='joined')
+    major_mine_application_document_xref = db.relationship(
+        'MajorMineApplicationDocumentXref',
+        lazy='select',
+        uselist=False,
+        primaryjoin='and_(MajorMineApplicationDocumentXref.mine_document_guid == MineDocument.mine_document_guid, MineDocument.is_archived == True)'
+    )
```

Comment:
I think there's a couple of more document types that would fall into this that can be archived (check out MineDocumentSearchUtil)
- ProjectSummaryDocumentXref
- ProjectDecisionPackageDocumentXref
- InformationRequirementsTableDocumentXref
- MajorMineApplicationDocumentXref

Reply:
Nice catch. I will implement them.

Reply:
@simensma-fresh updated as adviced.